enginetest/queries/script_queries.go

```diff
@@ -271,6 +271,38 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		// https://github.com/dolthub/dolt/issues/6682
+		Name: "display width for numeric types",
```

Comment:
Probably better in create_table_queries.go, trying not to expand these generic scripts any these days